owsi-core/owsi-core-examples/wicket-showcase/wicket-showcase-webapp/src/main/java/fr/openwide/core/showcase/web/application/portfolio/page/PortfolioMainPage.java

```diff
@@ -49,7 +51,7 @@ public PortfolioMainPage(PageParameters parameters) {
 		UserDataProvider userDataProvider = new UserDataProvider(searchTermModel, activeModel);
 		
 		// Porfolio
-		UserPortfolioPanel portfolioPanel = new UserPortfolioPanel("userPortfolio", userDataProvider, showcaseConfigurer.getPortfolioItemsPerPageUser());
+		UserPortfolioPanel portfolioPanel = new UserPortfolioPanel("userPortfolio", userDataProvider, ApplicationPropertyModel.of(ShowcaseWebappPropertyIds.PORTFOLIO_ITEMS_PER_PAGE_USER).getObject());
```

Comment:
I'm not sure what you mean here. <code>ApplicationPropertyModel.of()</code> already returns a IModel.